test/session.test.js

```diff
@@ -105,7 +104,7 @@ test('should allow get/set methods for fetching/updating session values', async
 test('should use custom sessionId generator if available (without request)', async (t) => {
   t.plan(2)
   const fastify = await buildFastify((request, reply) => {
-    t.equal(request.session.sessionId.startsWith('custom-'), false)
+    t.equal(request.session.sessionId.startsWith('custom-'), true)
```

Comment:
This is the only unit test i did change. I actually think that this is the correct behaviour.

Reply:
it seems that there was a race condition in the unit tests